build.gradle

```
@@ -204,7 +204,7 @@ def generate_distro_tasks = { dist, gDescription, gPackageName, gInputFile, gVer
           requires('adduser')
         }
 
-        suggests('default-jre | java11-runtime | java11-runtime-headless')
+        suggests('openjdk-17-jre |  java17-runtime  |  java17-runtime-headless')
```

```suggestion
        suggests('openjdk-17-jre | java17-runtime | java17-runtime-headless')
```

Perhaps it also make sense to add Zulu/Temurin JVM package names to the suggests for users on older distros (Debian 10) which do not hava Java 17 in their repos?

---

Good point!